README.md

```diff
@@ -193,3 +200,134 @@ For more information on running and tearing down k8s environments, follow this [
+kubectl config use-context docker-for-desktop  # Set the default context
+```
+
+### Helm Tiller
+
+Initialize the Helm Tiller:
+```bash
+helm init
+```
+
+### K8s Cluster Namespace
+
+As mentioned as part of the Anaxes Shipyard guidelines, you should deploy into a separate namespace in the cluster to avoid conflicts (create the namespace only if it does not already exist):
+```bash
+export DESIREDNAMESPACE=example
+kubectl create namespace $DESIREDNAMESPACE
+```
+
+This environment variable will be used in the deployment steps.
+
+## Deployment
+
+### 1. Add the Alfresco Helm repository to helm
+
+```bash
+helm repo add alfresco-incubator https://kubernetes-charts.alfresco.com/incubator
+```
+
+### 2. Add local DNS
+
+Add Local DNS Entry for Host Machine (needed for JWT issuer matching). Be sure to specify an active network interface.  It is not always ```en0``` as illustrated.  You can use the command ```ipconfig -a``` to find an active interface.
+
+```bash
+sudo sh -c 'echo "`ipconfig getifaddr en0`       localhost-k8s" >> /etc/hosts'; cat /etc/hosts
+```
+
+*Note:* If your IP address changes you will need to update the ```/etc/hosts``` entry for localhost-k8s.
+
+### 3. Deploy the DBP
+
+```bash
+helm install alfresco-incubator/alfresco-dbp \
+--set alfresco-infrastructure.alfresco-api-gateway.keycloakURL="http://localhost-k8s/auth/" \
+--set alfresco-infrastructure.rabbitmq-ha.enabled=false \
+--set alfresco-infrastructure.alfresco-activiti-cloud-registry.enabled=false \
+--set alfresco-infrastructure.alfresco-api-gateway.enabled=false \
+--set alfresco-content-services.externalHost="localhost-k8s" \
+--set alfresco-content-services.networkpolicysetting.enabled=false \
+--set alfresco-content-services.repository.environment.IDENTITY_SERVICE_URI="http://localhost-k8s/auth" \
+--set alfresco-content-services.repository.replicaCount=1 \
+--set alfresco-content-services.repository.livenessProbe.initialDelaySeconds=420 \
+--set alfresco-content-services.pdfrenderer.livenessProbe.initialDelaySeconds=300 \
+--set alfresco-content-services.libreoffice.livenessProbe.initialDelaySeconds=300 \
+--set alfresco-content-services.imagemagick.livenessProbe.initialDelaySeconds=300 \
+--set alfresco-content-services.share.livenessProbe.initialDelaySeconds=420 \
+--set alfresco-content-services.repository.resources.requests.memory="2000Mi" \
+--set alfresco-content-services.pdfrenderer.resources.requests.memory="500Mi" \
+--set alfresco-content-services.imagemagick.resources.requests.memory="500Mi" \
+--set alfresco-content-services.libreoffice.resources.requests.memory="500Mi" \
+--set alfresco-content-services.share.resources.requests.memory="1000Mi" \
+--set alfresco-content-services.postgresql.resources.requests.memory="500Mi" \
+--set alfresco-process-services.processEngine.environment.IDENTITY_SERVICE_AUTH="http://localhost-k8s/auth" \
+--set alfresco-process-services.processEngine.environment.IDENTITY_SERVICE_ENABLED=true \
```

Comment:
--set alfresco-process-services.processEngine.environment.IDENTITY_SERVICE_ENABLED="true"
This set needs to have the value in quotes.

Reply:
@svidrascu When I add the quotes it breaks the authentication flow.  Without them, it works correctly.